### PR TITLE
Add the example file for Radon measures

### PR DESCRIPTION
**Pull request by adamtopaz**
Please let me know if you think anything else should be added to the example file!

### CODE REVIEW

File: src/examples/radon_measures.lean [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+`CHFPNG₁_to_CHFPNGₑₗ` is the obvious forgetful functor between these two categories.
+-/
+example : CompHausFiltPseuNormGrp₁ ⥤ CompHausFiltPseuNormGrp :=
+CHFPNG₁_to_CHFPNGₑₗ
+
+example (X : CompHausFiltPseuNormGrp₁) :
+  (CHFPNG₁_to_CHFPNGₑₗ.obj X : Type*) = X := rfl
+
+/- The condensed abelian group `ℳ_p(S)` is isomorphic to the condensed abelian group assocaited
+to the CompHaus-ly filtered pseudo n ormed group `S.Radon_png p`.
```
Comment (jcommelin) [inline src/examples/radon_measures.lean:61]:
```suggestion
/- The condensed abelian group `ℳ_p(S)` is isomorphic to the condensed abelian group associated
to the CompHaus-ly filtered pseudo normed group `S.Radon_png p`.
```

File: src/examples/radon_measures.lean [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+-/
+example (S : Profinite.{0}) (μ : S.Radon_png p) : C(S,ℝ) →L[ℝ] ℝ :=
+μ.1
+
+lemma auxiliary_lemma (S : Profinite.{0}) (μ : weak_dual ℝ C(S,ℝ)) (p c : ℝ≥0) :
+  μ.bdd p c ↔
+  ∀ (ι : Fintype.{0}) (e : ι → set S)
+    (I : indexed_partition e) (he : ∀ i, is_clopen (e i)),
+    ∑ i : ι, ∥ μ.1 (clopens.indicator ⟨e i, he i⟩) ∥₊^(p : ℝ) ≤ c :=
+begin
```
Comment (jcommelin) [inline src/examples/radon_measures.lean:82]: Should this whole proof stay in the examples file? Or shall we hide it in a prereq file?

File: src/Radon/defs.lean [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+-/
+def Radon_LC_functor (p c : ℝ≥0) [fact (0 < p)] :
+  Profinite.{0} ⥤ Top.{0} :=
+{ obj := λ X, X.Radon_LC p c,
+  map := λ X Y f, map_Radon_LC f _ _,
+  map_id' := λ X,
+    by { ext, dsimp [map_Radon_LC, weak_dual.comap], congr' 1, ext, refl },
+  map_comp' := λ X Y Z f g, by { ext, refl } }
+
+.
```
Comment (adomani) [inline src/Radon/defs.lean:90]:
```suggestion
```
Maybe this is no longer needed?

File: src/examples/radon_measures.lean (hunk 2 above)
Comment (adamtopaz) [reply src/examples/radon_measures.lean]: Moved!

File: src/examples/radon_measures.lean [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+noncomputable theory
+
+open_locale liquid_tensor_experiment nnreal zero_object big_operators
+open liquid_tensor_experiment category_theory category_theory.limits
+  opposite pseudo_normed_group topological_space
+
+variables (p : ℝ≥0) [fact (0 < p)] [fact (p ≤ 1)]
+
+/-
```
[the remark this answers is not in the document]
Comment (adamtopaz) [reply src/examples/radon_measures.lean:18]: I don't think examples can (or should) have docstrings.

File: src/challenge_notations.lean [hunk 5]
```diff
@@ -31,7 +31,10 @@ tactic.extract_facts >>
   linarith]
 end
 
-localized "notation `ℳ_{` p' `}` S := (@real_measures.condensed p' _ (by fact_arith)).obj S"
+meta def tactic.interactive.fact_arith₂ :=
```
[the remark this answers is not in the document]
Comment (adamtopaz) [reply src/challenge_notations.lean:34]: Fixing this now....

[the remark this answers is not in the document]
Comment (adamtopaz) [reply src/challenge_notations.lean]: Done

File: src/examples/radon_measures.lean (hunk 4 above)
[the remark this answers is not in the document]
Comment (adomani) [reply src/examples/radon_measures.lean:18]: Ok, I tried and it seems that examples can have doc-strings, but maybe they are not needed, as you say.

**Review by jcommelin — APPROVED** [review]
Thanks for this massive effort!

**Pull request merged**